scripts/download-rc.js

```diff
@@ -43,7 +43,8 @@ const downloadRC = () => {
     if (fs.existsSync(`hazelcast-remote-controller-${HAZELCAST_RC_VERSION}.jar`)) {
         console.log('remote controller already exists, not downloading from maven.');
     } else {
-        console.log(`Downloading: remote-controller jar com.hazelcast:hazelcast-remote-controller:${HAZELCAST_RC_VERSION}`);
+        console.log('Downloading: remote-controller jar com.hazelcast:hazelcast-remote-controller:'
+        + HAZELCAST_RC_VERSION);
```

Comment:
```suggestion
            + HAZELCAST_RC_VERSION);
```

Reply:
I guess we should put a rule even for this if any. And I bet there is one. The goal of this pr is to detect these nits by linter.

Reply:
Could not find a rule for this. Also a rule that only forbids spaces in 
```
import { a} from b;
```

Reply:
Ok, changing this manually. Could not find a rule. I guess it does not exist. (I searched for aligning multiline string concatenations)